src/content_script/mastodonInject.js

```diff
@@ -0,0 +1,143 @@
+"use strict";
+
+const TIMEOUT_DURATION = 20000;
+
+/**
+ * Replacement onClick handler for Follow button.
+ * 
+ * @param {Event} event 
+ * @returns {void}
+ */
+function onClickFollow(event) {
+    event.stopPropagation();
+    event.preventDefault();
+    const username = window.location.pathname.split("/").slice(-1)[0];
+    // activate AutoRemoteFollow
+    window.open(`/users/${username}/remote_follow`, "_blank");
+}
+
+/**
+ * Replacement onClick handler for interaction buttons.
+ * 
+ * @param {Event} event
+ * @returns {void}
+ */
+function onClickInteract(event) {
+    event.stopPropagation();
+    event.preventDefault();
+    const articleElement = event.target.closest("article[data-id]");
+    const tootId = (
+        articleElement === null
+            ? window.location.pathname.split("/").slice(-1)[0]
+            : articleElement.getAttribute("data-id")
+    );
+    // activate AutoRemoteFollow
+    window.open(`/interact/${tootId}`, "_blank");
+}
+
+/**
+ * Wait for element to appear.
+ * 
+ * @param {string} selector
+ * @param {boolean} multiple
```

Comment:
```suggestion
 * @param {boolean} multiple optional, defaults to false
```

Or was there some jsdoc syntax for optional params, I don't recall.